media-libs/mlt/mlt-6.18.0.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="An open source multimedia framework"
+DESCRIPTION="A LGPL multimedia framework designed for TV broadcasting"
+HOMEPAGE="https://www.mltframework.org/"
+COPYRIGHT="2011-2019 by Meltytech, LLC."
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/mltframework/mlt/releases/download/v$portVersion/mlt-$portVersion.tar.gz"
+CHECKSUM_SHA256="fd2a346e0cc093e5d813e04cd126c077a2d224ae4d17b5226ff7478feaa445ea"
+PATCHES="mlt-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	mlt$secondaryArchSuffix = $portVersion
+	cmd:melt
+	lib:libmlt$secondaryArchSuffix = $libVersionCompat
+	lib:libmlt++$secondaryArchSuffix = $libVersionCompat
```

Comment:
libVersionCompat is wrong here, for libmlt it is 6, for libmlt++ it's 3 (same for devel)